test/testcases.py

```diff
@@ -68,7 +74,11 @@ def gen_testcases(what):  # pylint: disable=too-many-return-statements
     if what == "ami-boot":
         return [TestCaseCentos(image="ami"), TestCaseFedora(image="ami")]
     if what == "anaconda-iso":
-        return [TestCaseCentos(image="anaconda-iso"), TestCaseFedora(image="anaconda-iso")]
+        return [
+            klass(image="anaconda-iso", sign=sign, local=False)
+            for klass in (TestCaseCentos, TestCaseFedora)
+            for sign in [True, False]
```

Comment:
The iso tests are already pretty slow because of the depsolving/rpms involved. While this is a perfect test in the sense that it replicates exactly the issue we had and therefore validates the fix I do wonder if we could get away with a "cheaper" test. Maybe not but I think it's worth thinking about it .)

Reply:
Honestly, I am not sure we can test this without building the anaconda-iso image and the same will apply to the other disk images in the future. Maybe we can limit the test to only a subset of them: build just an ISO and one disk image for only one OS. This will test both paths (the one taken by anaconda-iso by calling directly `org.osbuild.skopeo stage` and the disk image path by calling `org.osbuild.container-deploy`). The next obvious question is: which OS should we choose?

Reply:
I have limited the tests to Fedora 40 ISO only